sections/collage.liquid

```diff
@@ -191,6 +193,34 @@
       "default": "collage",
       "label": "t:sections.collage.settings.mobile_layout.label"
     },
+    {
+      "type": "select",
+      "id": "color_scheme",
+      "options": [
+        {
+          "value": "accent-1",
+          "label": "t:sections.all.colors.accent_1.label"
+        },
+        {
+          "value": "accent-2",
+          "label": "t:sections.all.colors.accent_2.label"
+        },
+        {
+          "value": "background-1",
+          "label": "t:sections.all.colors.background_1.label"
+        },
+        {
+          "value": "background-2",
+          "label": "t:sections.all.colors.background_2.label"
+        },
+        {
+          "value": "inverse",
+          "label": "t:sections.all.colors.inverse.label"
+        }
+      ],
+      "default": "background-1",
+      "label": "t:sections.all.colors.label"
+    },
     {
```

Comment:
Here the image block still has a color scheme setting that needs to be removed. I'm doing it in one of my PR making sure the video and image block are using the card styling. So maybe no need to do it here 🤷

Reply:
I can wait until your collage PR merges with that removal just to ensure this is all behaving as expected.